pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -593,7 +599,83 @@ func (p *MasqueradePodInterface) createNatRules() error {
 			return err
 		}
 	}
-	return err
+
+	return nil
+}
+
+func (p *MasqueradePodInterface) createNatRulesUsingNftable() error {
+	err := Handler.NftablesLoad("ipv4-nat")
+	if err != nil {
+		return err
+	}
+
+	err = Handler.NftablesNewChain("nat", "KUBEVIRT_PREINBOUND")
```

Comment:
are all the functions here idempotent? meaning if "createNatRulesUsingNftable()" happens to get called multiple times during startup, will this logic not cause any errors?

Reply:
yes you can call this as many times as you want without returning any error.